spec/spec_helper.rb

```diff
@@ -1,7 +1,9 @@
-require 'simplecov'
-SimpleCov.start 'rails' do
-  add_filter '/config/'
-  add_filter '/lib/rspec/formatters/user_flow_formatter.rb'
+if ENV['CI']
```

Comment:
Right but Simplecov does work locally as well. For example if I made a new PR and it dropped out code coverage too much, I'd want to be able to run this locally to get the code coverage results generated. In that case, I think as a developer it would make more sense for me to turn on the COVERAGE feature, rather than the CI feature?

Again, not a huge deal, happy to merge this as-is to get the speedup changes